raycast/main.py

```diff
@@ -495,10 +505,6 @@ def pre_update(self):
     def update(self, events, dt):
         if self.state is None:
             self.state = self._build_initial_state()
-        if self.get_tick() % 20 == 0:
-            dims = self.get_screen_size()
-            cap = "Raycaster (DIMS={}, FPS={:.1f})".format(dims, self.get_fps(logical=False))
-            pygame.display.set_caption(cap)
```

Comment:
Running in the web context creates performance issues. The removal of instructions that are _optional_ (the fps value is already displayed via text) but have a small overhead in the web context seems to be a reasonable action. 

Indeed the `get_tick`, `get_screen_size`, `get_fps`, `format` and `set_caption` methods **will all be called** once per frame and consume some computation power in the web context, but in the web context this actually does nothing (the `set_caption` method has no possible implementation in the web context, therefore it does nothing) so some resource is wasted.

But if you consider this block to be an important part of the program,
another good way to handle this would be to use a flag "do we run in the web context?" 
One would initialize it at the very beginning (so we don't compute something irrelevant in the web context, at every frame) and wrap this block you have written within a
 `if self._is_web_context:` statement… _Do you prefer this solution?_ I can rewrite